packages/edit-site/src/components/page-pages/index.js

```diff
@@ -270,14 +259,8 @@ export default function PagePages() {
 			{
 				header: __( 'Date' ),
 				id: 'date',
+				type: DATE_TYPE,
```

Comment:
This could be extracted to a separate PR (rendering dates based on the `type`) as it's working well. Given it'd be only for this type, I don't see the point of doing it right now.